### PR TITLE
fix(react-components): cogs fixups and bump to 0.60.2

### PR DESCRIPTION
**Pull request by haakonflatval-cognite**
#### Type of change

Chore 

#### Jira ticket :blue_book:

https://cognitedata.atlassian.net/browse/

## Description :pencil:

Various format fixes related to the cogs-v10 migration

## Checklist :ballot_box_with_check:

- [x] I am proud of this feature.
- [ ] I have performed a self-review of my own code.
- [x] I have added PR type (Feat, Bug, Chore, Test, Docs, Style, Refactor) to the PR title.
- [ ] I have manually tested this for different scenarios (different models, formats, devices, browsers).
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the public documentation.
- [ ] I have added unit and visuals tests to prove that my feature works to the best of my ability.
- [ ] I have refactored the code for readability to the best of my ability.
- [ ] I have checked that my changes do not introduce regressions in the public documentation.
- [ ] I have outlined any known defects / lacking capabilities in the contents of this PR.
- [ ] I have listed any remaining work that should follow this PR in the description or jira/miro/etc.
- [ ] I have added TSDoc to any public facing changes.
- [ ] I have added "developer documentation" in any package README.md that is related / applicable to this PR.
- [ ] I have noted down and am currently tracking any technical debt introduced in this PR.
- [ ] I have thoroughly thought about the architecture of this implementation.
- [ ] I have asked peers to test this feature.

### CODE REVIEW

File: react-components/src/architecture/base/commands/SettingsItem.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+import { type BaseCommand } from './BaseCommand';
+
+export type SettingsItem = {
+  key: () => number;
+};
+
+export class CommandSettingsItem implements SettingsItem {
```
Comment (nilscognite) [inline react-components/src/architecture/base/commands/SettingsItem.ts:10]: I think this is not necessary and overcomplicated. All commands has their own unique id as a getter. You simple make a DividerCommand from BaseCommand, and override a minimum. Should be about nothing. Then you have everything you need, including the unique number

**Review by nilscognite — COMMENTED** [review]
Fix the SettingsItem and make the option button as they where before

**Review by nilscognite — CHANGES_REQUESTED** [review]
I simply don't understand that the new redesign option in the Settings panel is better than it was.

File: react-components/src/architecture/base/concreteCommands/SettingsCommand.ts [hunk 2]
```diff
@@ -20,7 +21,9 @@ export class SettingsCommand extends BaseSettingsCommand {
     super();
 
     this.add(new SetQualityCommand());
+    this.add(new PointCloudDividerCommand());
     this.add(new SetPointSizeCommand());
+    this.add(new PointCloudDividerCommand());
```
[the remark this answers is not in the document]
Comment (haakonflatval-cognite) [reply react-components/src/architecture/base/concreteCommands/SettingsCommand.ts]: bg0
It looks like this now, after I removed the second divider. I was worried it wouldn't be clear that the slider was associated to the label above, and not below. But I think it's reasonably clear, looking at it now. I'll keep it like this

[the remark this answers is not in the document]
Comment (nilscognite) [reply react-components/src/architecture/base/concreteCommands/SettingsCommand.ts]: The post is 

**Point settings:**
Size:      
Color:
Shape:
Filter:

**Review by nilscognite — APPROVED** [review]
LGTM

**Pull request merged**